src/embed_tests/Python.EmbeddingTest.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>net472;net6.0</TargetFrameworks>
+    <RollForward>LatestMajor</RollForward>
```

Comment:
This allows anyone that has recent versions of .net to run these tests. I can revert but I think its usefull, otherwise one needs to install .net6 which is already deprecated

Reply:
I think this should be conditioned on not running in CI

Reply:
Is there any config/env var which I can check?